{{ cookiecutter.dir_name }}/buildSrc/src/main/groovy/VocAndroidPlugin.groovy

```diff
@@ -0,0 +1,49 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+
+class VocPlugin implements Plugin<Project> {
+    void apply(Project project) {
+        project.extensions.create("voc", VocPluginExtension)
+
+        project.task('vocBuild') {
+            doLast {
+                def jarPath = "${project.voc.buildFromSourceDir}/dist/android/libs/python-android-support.jar"
```

Comment:
I don't think this part is needed; we can rely on `python-android-support.jar` being placed in the libs directory, either manually, or by briefcase.